x-pack/plugins/data_enhanced/common/search/session/types.ts

```diff
@@ -19,7 +19,12 @@ export interface BackgroundSessionSavedObjectAttributes {
   urlGeneratorId: string;
   initialState: Record<string, unknown>;
   restoreState: Record<string, unknown>;
-  idMapping: Record<string, string>;
+  idMapping: Record<string, BackgroundSessionSearchInfo>;
+}
+
+export interface BackgroundSessionSearchInfo {
```

Comment:
//nit 
`BackgroundSessionInfo` sounds sufficient

Reply:
(I think we're going to rename the feature to SearchSessions soon, so SearchSessionInfo also sounds decent)

Reply:
I wanted to include `search` in the name somehow because it's for a specific search request, not something on the entire session. Okay if we rename in the PR that we rename to `SearchSession`?